src/Http/Routing.Abstractions/src/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,10 @@
 [assembly: TypeForwardedTo(typeof(IEndpointFeature))]
 [assembly: TypeForwardedTo(typeof(IRouteValuesFeature))]
 [assembly: TypeForwardedTo(typeof(Endpoint))]
+// This warning cannot be suppressed in the Enumerator source because constructor is implicit.
+#pragma warning disable RS0016 // Symbol 'implicit constructor for 'Enumerator'' is not part of the declared API.
 [assembly: TypeForwardedTo(typeof(EndpointMetadataCollection))]
+#pragma warning restore RS0016 // Symbol 'implicit constructor for 'Enumerator'' is not part of the declared API.
```

Comment:
Need for this looks like an analyzer bug. I seem to remember it's been fixed in more recent version of the package. But, switching versions is a larger change because it requires deeper changes in how we reference the package.

Reply:
Why does it need to be suppressed at all? Why not just add the constructor to the PublicAPI.Unshipped.txt?